runtime/queries/sql/highlights.scm

```diff
@@ -24,20 +24,20 @@
 (term
   alias: (identifier) @variable.parameter)
 
-(term
+((term
    value: (cast
     name: (keyword_cast) @function.builtin
-    parameter: [(literal)]?))
+    parameter: [(literal)]?)))
```

Comment:
I don't think these parentheses have any function, but I copied them from upstream to have as little difference as possible, makes `diff` updates easier

(same for the extra leading space a few lines down)